src/components/sections/hero.section.tsx

```diff
@@ -8,30 +8,32 @@ interface HeroSectionProps {
 }
 
 export const HeroSection: FunctionComponent<HeroSectionProps> = ({ motto, description, subtitle }) => (
-  <Flex flexDirection="column" alignItems="center" mt={10} mb={6}>
-    <Flex flexDirection={['column', 'column', 'column', 'row']}>
+  <Flex flexDirection="column" alignItems="center" mt={[7, 10]} pt={7} mb={[2, 5]} mx={[0, 0, 0, 6]}>
+    <Flex flexDirection={['column', 'column', 'column', 'row']} mb={[3, 7]}>
       <Box>
         <H1 color="primary.dark">{motto}</H1>
       </Box>
-      <Flex flexDirection="column">
+      <Flex flexDirection="column" mt={[0, 0, 0, 4]}>
         <Box>
           <H4 color="grey.dark.3">{description}</H4>
-          <Flex flexDirection="row">
+          <Flex flexDirection={['column', 'row']}>
             <TextInput
               placeholder="Enter your e-mail"
               help='By clicking "try out!" your agree to our Privacy Policy.'
               icon={<EmailIcon />}
             ></TextInput>
```

Comment:
ide nem kell lezaro tag, ha nincs kozte semmi `children`
```suggestion
            />
```

Reply:
prettier néha beteszi így, nem tudom mi alapján csinálja.. nincs erre valami beállítás?

Reply:
szerintem nem a prettier teszi be, hanem valami masik tool, ami automatikusan beilleszti a lezaro taget